<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTreeView" name="treeView">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTextEdit" name="noteEditor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuSwitchProfile">
     <property name="title">
      <string>Switch profile</string>
     </property>
    </widget>
    <addaction name="actionNewNotebook"/>
    <addaction name="actionAddNotebook"/>
    <addaction name="actionDeleteNotebook"/>
    <addaction name="separator"/>
    <addaction name="actionDeleteNote"/>
    <addaction name="actionSaveNote"/>
    <addaction name="separator"/>
    <addaction name="actionProfileManager"/>
    <addaction name="menuSwitchProfile"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionFeedback"/>
    <addaction name="separator"/>
    <addaction name="actionAboutSturdy"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNewNotebook">
   <property name="text">
    <string>&amp;New note</string>
   </property>
  </action>
  <action name="actionAddNotebook">
   <property name="text">
    <string>Add notebook</string>
   </property>
  </action>
  <action name="actionDeleteNotebook">
   <property name="text">
    <string>Delete notebook</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionAboutSturdy">
   <property name="text">
    <string>About &amp;Sturdy</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionSaveNote">
   <property name="text">
    <string>Save note</string>
   </property>
  </action>
  <action name="actionDeleteNote">
   <property name="text">
    <string>Delete note</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="actionProfileManager">
   <property name="text">
    <string>Profile manager</string>
   </property>
  </action>
  <action name="actionFeedback">
   <property name="text">
    <string>Submit &amp;Feedback</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
